Maven.md

```diff
@@ -129,20 +129,20 @@ The scripts that fetch build scan data expect some of it to be present as custom
 ## Redirecting build scan publishing
 
 The scripts that run one or more builds locally can be configured to publish build scans to a different
-Gradle Enterprise server than the one that the builds point to by passing the `-s` or `--gradle-enterprise-server`
+Develocity server than the one that the builds point to by passing the `-s` or `--gradle-enterprise-server`
 command line argument. In the example below, the script will configure the local builds to publish their build scans
 to ge.example.io regardless of what server is configured in the build.
 
 ```bash
 ./01-validate-local-build-caching-same-location.sh -i -s https://ge.example.io
```

Comment:
```suggestion
./01-validate-local-build-caching-same-location.sh -i -s https://develocity.example.io
```

Probably would be worth doing a find-replace on `ge.example.io`.

Reply:
Done!